controllers/childGuidance.js

```diff
@@ -0,0 +1,133 @@
+import childModel from '../models/childModel.js';
+
+const createGuidance = async (req, res) => {
+  try {
+    const data = req.body;
+    const check = await childModel.findOne({ guideId: data.guideId });
+    if (check) {
+      return res.status(409).json({
+        message: "Child guide already exists",
+      });
+    }
+
+    const guideInstance = new childModel({
+      name: data.name,
+      age: data.age,
+      guides: [
+        {
+          question: "My child has a fever, what can I do?",
+          answer:
+            "If your child has a fever, you have to visit the closest hospital as soon as possible",
+          category: "Healthcare",
+        },
+      ],
+      guideId: data.guideId,
```

Comment:
- why are you hard coding the question, answer and the category? that was given to you as an example
- again, remove the guideId, this will be automatically created in mongodb